detection_rules/etc/stack-schema-map.yaml

```diff
@@ -99,28 +99,13 @@
 
 ## Supported
 
-"8.14.0":
-  beats: "8.14.3"
-  ecs: "8.11.0"
-  endgame: "8.4.0"
-
-"8.15.0":
-  beats: "8.15.3"
-  ecs: "8.11.0"
-  endgame: "8.4.0"
-
-"8.16.0":
-  beats: "8.16.1"
-  ecs: "8.16.0"
-  endgame: "8.4.0"
-
-"8.17.0":
-  beats: "8.17.0"
+"8.18.0":
+  beats: "8.18.0"
```

Comment:
Should we comment the old ones out instead of deleting?

Reply:
Done